docs/src/tutorials/tutorial_1.md

```diff
@@ -30,34 +36,43 @@ In order to run FUND, you will need to have the packages `Distributions` and `St
 
 Now open a julia REPL and type 
 
-```
+```jldoctest tutorial1; output = false
 using MimiFUND
-```
-to access the public API to FUND, which currently includes the function `getfund`, a function that returns a version of fund allowing for different user specifications.  
+
+# output
 
 ```
-m = getfund()
+to access the public API to FUND, which currently includes the function `MimiFUND.get_model` (which can also be called with the exported and identical function `getfund`). This function returns a version of fund allowing for different user specifications.  
+
+```jldoctest tutorial1; output = false
+m = MimiFUND.get_model()
 run(m)
+
+# output
+
 ```
 
-Note that these steps should be relatively consistent across models, where a repository for `ModelX` should contain a primary file `ModelX.jl` which exports, at minimum, a function named something like `getModelX` or `construct_ModelX` which returns a version of the model, and can allow for model customization within the call.
+Note that these steps should be relatively consistent across models, where a repository for `ModelX` should contain a primary file `ModelX.jl` which exports, at minimum, a function named something like `get_model` or `construct_model` which returns a version of the model, and can allow for model customization within the call.
 
-In this case, the function `getfund` has the signature
-``` 
-getfund(; nsteps = default_nsteps, datadir = default_datadir, params = default_params)
+In this case, the function `get_model` has the signature
+```julia
+MimiFUND.get_model(; nsteps = default_nsteps, datadir = default_datadir, params = default_params)
 ```
 Thus there are no required arguments, although the user can input `nsteps` to define the number of timesteps (years in this case) the model runs for, `datadir` to define the location of the input data, and `params`, a dictionary definining the parameters of the model.  For example, if you wish to see only the first 100 timesteps,you may use:
-```
+```julia
 using MimiFUND
-m = getfund(nsteps = 100)
+m = MimiFUND.get_model(nsteps = 100)
```

Comment:
@ckingdon95 I'm seeing an error after this call now:

```
ERROR: Mismatched data size for a parameter connection: dimension :time in <ComponentId MimiFUND.scenariouncertainty> has 101 elements; external parameter :scenpgrowth has 1051 elements.
```

any ideas?

Reply:
This PR address this and needs discussion:
https://github.com/fund-model/MimiFUND.jl/pull/44

Reply:
Ah ok thanks!

Reply:
@ckingdon95 I’m realizing this actually might work in real cases since it’s case 2 (the user set a shorter parameters dictionary) ... I have that in this code but don’t run that reset through the doctests code blocks because I don’t have an actual shorter set of parameters on hand.  If I’d done that I bet it would’ve worked.

Reply:
@lrennels should we change this `nsteps` example since the fix for this isn't merged into FUND yet? Or should we wait? Ideally these blocks would also be doctested right?

Reply:
I think you’re right we should change or remove it since it isn’t merged yet and we’d want these blocks doctested.